CHANGES.md

```diff
@@ -3,8 +3,11 @@ Change Log
 
 ## TODO release
 
-* Added `clippingPlanes` property to `ModelGraphics` and `Cesium3DTileset`, which specifies an array of planes to clip the object. [TODO]()
+* Added `clippingPlanes` property to `ModelGraphics` and `Cesium3DTileset`, which specifies a `ClippingPlanesCollection` to clip the object. [TODO]()
```

Comment:
Update the TODO links?  I think we can merge `clip-planes-master` into `master` after this PR is merged, right?